skietbaan_fe/src/components/ViewMembers.css

```diff
@@ -7,35 +7,131 @@
     }
   }
 
-  body{
-    background-color:red;
-  }
-
   .col-1,.col-2,.col-3,.col-4,.col-5,.col-6,.col-7,.col-8,.col-9,.col-10,.col-11,.col-12{
     float:left;
   }
 
   .centre-view-member{
     margin:0 auto;
     position: center;
-    background-color: white;
   }
 
   .search-member-name{
     background-color: #171F32;
   }
 
   .table-search-members{
-    background-color: white;
+    background-color: #171F32;
   }
 
-  .page-name{
+  .page-name-view{
     position: center;
     text-align: center;
+    text-size-adjust: 25px;
+    background-size: 15px;
+    height: 49px;
+    background-color: #0c0c0c;
+  }
+
+  .go-back-to-create-page{
+    float: left;
+    padding-top: 10px;
+    padding-left: 10px;
+    background-size: 15px;
+    height: 40px;
   }
 
   .label-search{
     position: center;
     text-align: center;
     color: white;
   }
+
+  .username-search{
+    position: center;
+    text-align: center;
+  }
+
+  .userValue {
```

Comment:
inconsistent indentation and please leave a space after classname and first bracket this is also inconsistent

Reply:
made it look better